test/benchmark/bench_linux_and_aster.sh

```diff
@@ -44,6 +44,13 @@ prepare_libs() {
     fi
 }
 
+# Prepare Linux initrd
+prepare_initrd() {
```

Comment:
This function name is inappropriate since mke2fs is not about initrd. Rename this function to `prepare_fs`.